package.json

```diff
@@ -26,11 +29,10 @@
     ]
   },
   "scripts": {
-    "postinstall": "(cd api && echo 'Installing API node modules' && npm i); (cd jigsaw && echo 'Installing jigsaw node modules' && npm i); (cd authorizer && echo 'Installing Authorizer node modules' && npm i);",
+    "postinstall": "(cd api && echo 'Installing API node modules' && npm i); (cd jigsaw && echo 'Installing jigsaw node modules' && npm i);",
```

Comment:
Again not related to this change in particular, but have we considered/tried something like [Lerna](https://lerna.js.org/) for dependency management in monorepos? Not that Lerna is perfect, but might avoid some of this dependency install wizardry.